tracks/go/exercises/nucleotide-count/mentoring.md

```diff
@@ -0,0 +1,98 @@
+# Nucleotide Count
+
+### Reasonable Solutions
+
+Using a map is probably the most idiomatic solution to this problem, especially since the Histogram type basically must be a map. Using a switch is about 4 times faster over the provided test cases because it does not require hashing.
```

Comment:
```suggestion
Using a map is probably the most idiomatic solution to this problem, especially since the Histogram type basically must be a map.
Using a switch is about four times faster over the provided test cases because it does not require hashing.
```

Exercism uses one sentence per line. 

> four times faster over the provided test cases 

I'm not sure what this means. Can you reword this?